### DISCUSSION TITLE
Autograding results (download) do not include the student's registered e-mail address

### DISCUSSION THREAD

**Discussion by LuisDamas** [community/community#112639] [GitHub Education]
### Hi everyone,

We are using github autograding with students from different courses and asking them to register via institutional email.
However, downloading the automatic assessment results generates a csv file with the github username, not the email that could identify the student and their grade.
Is it possible to (also) download the email address registered when downloading the autograding results?

**Comment by RyanHecht** [comment]
Hi Luis,
To solve this problem I recommend that you utilize Classrooms [roster feature](https://docs.github.com/en/education/manage-coursework-with-github-classroom/teach-with-github-classroom/manage-classrooms#about-classroom-rosters) to link student names/emails to GitHub identifiers. When you download a CSV the roster there will then be a `roster_identifier` column
